### ISSUE TITLE
Code doesn't run

### ISSUE THREAD

**Issue by oneshotbob** [alpacahq/example-scalping#16]
When I run the code exactly as is I get the following message

2021-03-30 08:58:25,672:rest.py:142:WARNING:alpaca_trade_api.rest:sleep 3 seconds and retrying https://data.alpaca.markets/v2/stocks/TSLA/bars 3 more time(s)...

The message just repeats over and over again and nothing happens.

**Comment by bibib2211** [comment]
same here

**Comment by bjacobo20** [comment]
yup same here. I did notice that if you use api.get_quotes instead of api.get_bars, it doesnt show that error. But of course, the code would need to be changed to use bar data.

**Comment by juanrossxo** [comment]
Same here.. Anyone get a fix? Odd that Alpaca publishes these cool articles and doesn't have workable code with v2 API..

**Comment by rahulmr** [comment]
Same issue how to fix this??

**Comment by camelpac** [comment]
Hi guys,
this code example doesn't work with the free accounts since in the free account you have a 15 minute delay (so you cannot do scalping).

**Comment by babu2306** [comment]
is this code working?...

**Comment by alphadog2015** [comment]
Is anyone running this code succesfully?

Thanks

**Comment by hriday2771** [comment]
The code does not run even with the Pro subscription, please help
